Sources/Winget-AutoUpdate/functions/Update-App.ps1

```diff
@@ -33,6 +33,10 @@ Function Update-App ($app) {
         Write-ToLog "-> Running (overriding default): Winget upgrade --id $($app.Id) -e --accept-package-agreements --accept-source-agreements -s winget --override $ModsOverride"
         & $Winget upgrade --id $($app.Id) -e --accept-package-agreements --accept-source-agreements -s winget --override $ModsOverride | Where-Object { $_ -notlike "   *" } | Tee-Object -file $LogFile -Append
     }
+    elseif ($ModsCustom) {
+         Write-ToLog "-> Running (customizing default): Winget upgrade --id $($app.Id) -e --accept-package-agreements --accept-source-agreements -s winget -h --custom $ModsCustom"
+         & $Winget upgrade --id $($app.Id) -e --accept-package-agreements --accept-source-agreements -s winget --custom $ModsCustom | Where-Object { $_ -notlike "   *" } | Tee-Object -file $LogFile -Append
```

Comment:
**-h** is missing from the command even if it writes to log that it's used.
On purpose?

Reply:
My fault. Thank you!